.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-env
\ No newline at end of file
+serverlog
+userlog
+.env
```

Comment:
[OPTIONAL] You can leave the `.env` file in the remote repository so we can always work on the same localhost PORT.